addons/stock_account/models/product.py

```diff
@@ -135,8 +210,6 @@ class ProductProduct(models.Model):
         help="Technical field to correctly show the currently selected company's currency that corresponds "
              "to the totaled value of the product's valuation layers")
     stock_valuation_layer_ids = fields.One2many('stock.valuation.layer', 'product_id')
-    valuation = fields.Selection(related="categ_id.property_valuation", readonly=True)
-    cost_method = fields.Selection(related="categ_id.property_cost_method", readonly=True)
```

Comment:
should be converted to computed field to get the company value if no categ

Reply:
https://github.com/odoo/odoo/pull/166323#discussion_r1806814856 It will get field values from template via inherits